css/templates/themes/2/page.css

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2017 The AMPStart Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../../core';
+@import '../../../../components/footer/footer';
+@import '../../../../components/images/fullpage-hero';
+@import '../../../../components/images/images';
+@import '../../../../components/navbar/navbar';
+@import '../../../../components/social-follow/social-follow';
+@import 'page-vars';
+
+body {
+  line-height: 1.75;
+}
+
```

Comment:
may be consider rewriting this scale in your config so that you dont have to do this (look at `bass-override-vars`)

``` css
  --line-height-1: 1rem;
  --line-height-2: 1.125rem;
  --line-height-3: 1.5rem;
  --line-height-4: 2rem;
```